### PR TITLE
Market Hours Database Adjustment

### LINKED ISSUE #4559: Fix 4th July Error in MHDB
#### Expected Behavior
MHDB Accurate

#### Actual Behavior
Missed observed holiday on 3rd July 2020
https://github.com/QuantConnect/Lean/blob/master/Data/market-hours/market-hours-database.json#L252

#### Potential Solution
Review all holidays for observed dates. 

 
#### Checklist 
- [x] I have completely filled out this template
- [x] I have confirmed that this issue exists on the current `master` branch
- [x] I have confirmed that this is not a duplicate issue by searching [issues](https://github.com/QuantConnect/Lean/issues)

- [x] I have provided detailed steps to reproduce the issue

### PR DESCRIPTION
**Pull request by C-SELLERS**
#### Description

Adjust Market hours database entries for some mistakes found

#### Related Issue

Closes #4559 

#### Motivation and Context

Bad dates in database

#### Requires Documentation Change

None

#### How Has This Been Tested?

N/A

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (non-breaking change which improves implementation)
- [ ] Performance (non-breaking change which improves performance. Please add associated performance test and results)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Non-functional change (xml comments/documentation/etc)

#### Checklist:

- [x] My code follows the code style of this project.
- [x] I have read the **CONTRIBUTING** [document](https://github.com/QuantConnect/Lean/blob/master/CONTRIBUTING.md).
- [x] I have added tests to cover my changes. 
- [x] All new and existing tests passed.
- [x] My branch follows the naming convention `bug-<issue#>-<description>` or `feature-<issue#>-<description>`

### CODE REVIEW

File: Data/market-hours/market-hours-database.json [hunk 1]
```diff
@@ -356,6 +356,7 @@
         "11/29/2019": "13:00:00",
         "11/27/2020": "13:00:00",
         "11/26/2021": "13:00:00",
+        "11/25/2022": "13:00:00",
         "12/24/2009": "13:00:00",
         "12/24/2012": "13:00:00",
```
Comment (Martin-Molinero) [inline Data/market-hours/market-hours-database.json:361]: There is another `"7/4/2020"` around line 26673, should it also be fixed? There are a couple of other `"12/25/2022"` too

Comment (C-SELLERS) [reply Data/market-hours/market-hours-database.json:361]: Options dates are updated! I verified as many of the major entries dates as I could, some of the distant dates did not have calendars to verify, but nothing seemed out of the ordinary with alignment with previous years.

Comment (Martin-Molinero) [reply Data/market-hours/market-hours-database.json:361]: Just checking: I see 10 matches of `12/25/2022`, here we are only changing 2, that's okay?

Comment (C-SELLERS) [reply Data/market-hours/market-hours-database.json:361]: Yes the other 8 entries are for markets that are open everyday not weekday markets. They all have Christmas closed on the actual day.

**Review by Martin-Molinero — APPROVED** [review]

**Review by jaredbroad — APPROVED** [review]
Thank you @C-SELLERS

**Pull request merged**